--- mozilla/extensions/enigmail/ui/content/contents.rdf.orig	2004-09-17 10:52:05.000000000 +0000
+++ mozilla/extensions/enigmail/ui/content/contents.rdf	2004-09-17 10:52:15.000000000 +0000
@@ -13,7 +13,7 @@
         chrome:author="Team Enigmail"
         chrome:authorURL="http://enigmail.mozdev.org/"
         chrome:name="enigmail"
-        chrome:extension="true"
+        chrome:extension="false"
         chrome:description="This extension adds support for OpenPGP message encryption."
         chrome:settingsURL="chrome://enigmail/content/pref-enigmail.xul">
      <chrome:uninstallInfo>
