--- ./comm/mail/app/profile/all-thunderbird.js.orig	2014-03-18 23:18:29.607626995 +0000
+++ ./comm/mail/app/profile/all-thunderbird.js	2014-03-18 23:18:39.476627310 +0000
@@ -211,7 +211,7 @@
 pref("general.autoScroll", true);
 #endif
 
-pref("mail.shell.checkDefaultClient", true);
+pref("mail.shell.checkDefaultClient", false);
 pref("mail.spellcheck.inline", true);
 
 // Force the unit shown for the size of all folders. If empty, the unit
